src/main/java/edu/harvard/iq/dataverse/Dataverse.java

```diff
@@ -111,23 +111,23 @@ public void setDataverseType(DataverseType dataverseType) {
 
         public String getFriendlyCategoryName(){
        switch (this.dataverseType) {
-            case RESEARCHERS:
-                return BundleUtil.getStringFromBundle("dataverse.type.selectTab.researchers");
-            case RESEARCH_PROJECTS:
-                return BundleUtil.getStringFromBundle("dataverse.type.selectTab.researchProjects");
-            case JOURNALS:
-                return BundleUtil.getStringFromBundle("dataverse.type.selectTab.journals");           
-            case ORGANIZATIONS_INSTITUTIONS:
-                return BundleUtil.getStringFromBundle("dataverse.type.selectTab.organizationsAndInsitutions");           
-            case TEACHING_COURSES:
-                return BundleUtil.getStringFromBundle("dataverse.type.selectTab.teachingCourses");
-            case LABORATORY:
-               return BundleUtil.getStringFromBundle("dataverse.type.selectTab.laboratory");
-            case RESEARCH_GROUP:
-               return BundleUtil.getStringFromBundle("dataverse.type.selectTab.researchGroup");
-            case DEPARTMENT:
-                return BundleUtil.getStringFromBundle("dataverse.type.selectTab.department");
-            case UNCATEGORIZED:
+           case RESEARCHERS:
+               return BundleUtil.getStringFromBundle("dataverse.type.selectTab.researchers", null , "en");
```

Comment:
can we have this get the value from the default bundle and not hard coded "en"?

Reply:
@scolapasta  - committed new code. please verify